PingCAP-QE/Naglfar
# TestClusterTopology: support deploy flink cluster

### What problem does this PR solve?

Issue Number: close #92  

Problem Summary: supports deploy Flink cluster and interact with the deployed tidb cluster

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

How it Works:
1.add flink related fields in testclustertopology.types
2.pass in related parameters and cmd when initializing resources

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 


- Manual test (add detailed scripts or steps below)
1.deploy flink cluster by yaml
2.check taskmanager and jobmanager if is ok by webui 
Side effects

- Breaking backward compatibility
